go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -872,7 +872,7 @@ func (nc *DefaultNodeNetworkController) Start(ctx context.Context) error {
 						nodeSubnets, err := util.ParseNodeHostSubnetAnnotation(&node, types.DefaultNetworkName)
 						if err != nil {
 							err1 = fmt.Errorf("unable to fetch node-subnet annotation for node %s: err, %v", node.Name, err)
-							return false, nil
+							continue
```

Comment:
@jluhrsen while you are here, coudl you also fix line 888, 908 : 
```
klog.Infof("Upgrade Hack: Syncing nodes took %v", time.Since(start))
```

this always reports time since the loop started, not the time since the these things actually completed. In other words if the node check passed in 5 seconds, and then after this services are checked and fail, then the entire loop will restart, and now the line will print time a new longer time for how long it took to sync nodes.

Reply:
i see how this can be confusing :) i did this intentionally and would subtract the time printed from previous time print, but +1 to this comment to make this better.

Reply:
ok to just bump the `start := time.Now()` to the beginning of each PollUntilContextTimeout() ? that should reset it each time we work through the syncs